gateway/api/views.py

```diff
@@ -150,12 +157,18 @@ def run(self, request):
             if not serializer.is_valid():
                 return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+            try:
```

Comment:
If we replace it with

```python
jobconfig = None
config_data = request.data.get("config")
if config_data:
    config_serializer = JobConfigSerializer(data=json.loads(config_data))
    if not config_serializer.is_valid():
        return Response(config_serializer.errors, status=status.HTTP_400_BAD_REQUEST)

    jobconfig = config_serializer.save()
```

then we resolve validation + errors will be in a nice understandable format 

```
QuantumServerlessException: 
| Message: Http bad request.
| Code: 400
| Details: {"workers":["Ensure this value is less than or equal to 5."]}
```

and we will not longer need `save_jobconfig` function.